docs/build/collator/how-to-stake-in-zeitgeist.md

```diff
@@ -59,6 +59,16 @@ console.log(delegatorInfo.toHuman()["delegations"].length);
 
 ![](/img/get-your-delegations-number.png)
 
+### Check the amount you can stake
+
+You have a balance which consists of transferrable, vested, locked and reserved tokens. To see the amount of stakable tokens, you can check the transferable tokens on the ` Account` Page
```

Comment:
Strictly speaking the balances pallet only knows transferable, reserved and locked tokens. Vested tokens are locked tokens that can be unlocked but have not been unlocked yet.